<launch>
    <arg name="logoutput" default="screen"/>
    <rosparam command="delete" param="/RoverSerial" />
    <rosparam command="load" file="$(find rover_20_serial)/config/serial_config.yaml" />
    <node name="serial_rover" pkg="rover_20_serial" type="arm_serial.py" output="$(arg logoutput)"/>
    
<node pkg ="twist_mux" type ="twist_mux" name="twist_mux">
<rosparam command="load" file="$(find rover_20_serial)/config/twist_mux.yaml"/>
<remap from="cmd_vel_out" to="cmd_vel"/>
</node>
    
</launch>
